docs/packages/os.rst

```diff
@@ -1,10 +1,10 @@
 OS
 --
 
- * `Android-Apk <https://github.com/ruslo/hunter/wiki/pkg.android.apk>`_
- * `Android-Modules <https://github.com/ruslo/hunter/wiki/pkg.android.modules>`_
- * `Android-SDK <https://github.com/ruslo/hunter/wiki/pkg.android.sdk>`_
- * `ios_sim <https://github.com/ruslo/hunter/wiki/pkg.ios_sim>`_
- * `Qt-Android-CMake <https://github.com/ruslo/hunter/wiki/pkg.qt.android.cmake>`_
- * `Washer <https://github.com/ruslo/hunter/wiki/pkg.washer>`_ - Lightweight, header-only, C++ wrapper around the Windows API
- * `WTL <https://github.com/ruslo/hunter/wiki/pkg.wtl>`_ - Windows Template Library (WTL>`_ is a C++ library for developing Windows applications and UI components.
+ * `Android-Apk <https://docs.hunter.sh/en/latest/packages/pkg/Android-Apk.html>`_
```

Comment:
It should be local link:
```
:ref:`pkg.Android-Apk`
```

It will go to `.. _pkg.Android-Apk:` from file:
* https://raw.githubusercontent.com/ruslo/hunter/cbb01fcf9edb942e6e049b2648120dfc93abc88c/docs/packages/pkg/Android-Apk.rst